### PR TITLE
Masterbar: add title attribute to recent drafts button.

### PR DESCRIPTION
**Pull request by michaeldcain**
Adds a title to the drafts number for context around what I'm about to click.

draft-title

### CODE REVIEW

**Comment by mtias** [comment]
Looks good.

**Comment by marekhrabe** [comment]
Have you considered using a tooltip? I'm not sure users understand and expect titles to appear. I'm not suggesting it's a wrong solution, just wondering whether we can do even better.

screen shot 2017-03-10 at 16 31 41

File: client/layout/masterbar/drafts.jsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ class MasterbarDrafts extends Component {
 			<div>
 				<QueryPostCounts siteId={ selectedSite.ID } type="post" />
 				{ this.props.draftCount > 0 &&
-					<Button compact borderless className="masterbar__toggle-drafts" onClick={ this.toggleDrafts } ref="drafts">
+					<Button compact borderless className="masterbar__toggle-drafts" onClick={ this.toggleDrafts } ref="drafts" title={ translate( 'Latest drafts' ) }>
```
Comment (a8ci18n) [inline client/layout/masterbar/drafts.jsx]: Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Latest Drafts' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (marekhrabe) [inline client/layout/masterbar/drafts.jsx]: This bot is ❤️

**Comment by michaeldcain** [comment]
>Have you considered using a tooltip?
@marekhrabe: Since it queues a popover, a tooltip might feel redundant. I was also following the title attribute pattern of the other navigation items.

**Pull request merged**